helpers/errorCodes.js

```diff
@@ -70,6 +70,10 @@ module.exports = {
 		code: 'E0016',
 		message: 'Invalid parameters'
 	},
+	E0017: {
+		code: 'E0017',
+		message: 'Service Unavailable'
+	},
```

Comment:
added a new error code for services that are disabled. Do you guys approve? ill add it to resources documentation if you do otherwise ill revert this commit